manifests/mock/create_mock.pp

```diff
@@ -1,10 +1,11 @@
 define rpmbuilder::mock::create_mock (
-  $dist       = undef,
-  $release    = undef,
-  $proxy      = false,
-  $vendor     = "Puppet User",
-  $arch       = undef,
-  $mock_root  = "/etc/mock",
+  $dist         = undef,
+  $release      = undef,
+  $proxy        = false,
+  $vendor       = "Puppet User",
+  $arch         = undef,
+  $mock_root    = "/etc/mock",
+  $mirror_url   = undef,
```

Comment:
The aligning assignment operator thing makes it difficult to distinguish significant from insignificant changes :(